lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/handler/CheckPassportHandler.java

```diff
@@ -338,7 +338,9 @@ private boolean determineVerificationRetryStatus(
             DocumentDataVerificationResult documentDataVerificationResult,
             final int MAX_ATTEMPTS) {
 
-        if (documentDataVerificationResult.isVerified()) {
+        if (documentDataVerificationResult.isVerified()
+                && (documentDataVerificationResult.getContraIndicators() == null
```

Comment:
Null check not needed here as calculateContraIndicators() guarantees a non-null list.

https://github.com/alphagov/di-ipv-cri-uk-passport-api/blob/dc09169f81daf18e89e023fd223dfeb093e2bc4d/lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/services/DocumentDataVerificationService.java#L211

Reply:
We need it in order for the tests to pass as they still use nulls